mkosi/config.py

```diff
@@ -498,6 +499,33 @@ def match_systemd_version(value: str) -> bool:
     return config_match_version(value, version)
 
 
+def config_parse_bytes(value: Optional[str], old: Optional[int] = None) -> Optional[int]:
+    if not value:
+        return None
+
+    if value.endswith("G"):
+        factor = 1024 ** 3
+    elif value.endswith("M"):
+        factor = 1024 ** 2
```

Comment:
nit: Even black now respects this being formatted as
```suggestion
        factor = 1024**3
    elif value.endswith("M"):
        factor = 1024**2
```